tests/regression/tests/REQUEST-921-PROTOCOL-ATTACK/921151.yaml

```diff
@@ -62,6 +62,6 @@ tests:
               Host: "localhost"
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             data: '941151-4=test\r\nmore=text'
```

Comment:
I don't see the `uri` for this test (921151-4), should it be a `/post`?

Reply:
Rules in 920 and 921 are attacks directly on protocol and i'm not sure if missing uri is a purpose or not.

Reply:
That would default to `/`. I'd prefer to be explicit with the URI.